huxflux/huxflux.js

```diff
@@ -1,10 +1,28 @@
+
 /**
- * @returns {Number[]} Lista med sju nummer
+ * @returns {Number[]} list of seven integers
  */
 function huxflux() {
-    return; // Returnera din lista med nummer här
+
+    const Number = []
+   
+    while(Number.length<7){
+  
+      do{  //the do while loop will fill the Number array with random integers. 
+        var occuring = false  
+        var random = Math.floor(Math.random() * 35) + 1
```

Comment:
The variable type var on line 13 could be changed to the type const to align to ES6. If you are interested to know a bite about the difference you can read up about the different variabel types here. https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e#:~:text=var%20variables%20can%20be%20updated,be%20updated%20nor%20re%2Ddeclared.&text=While%20var%20and%20let%20can,must%20be%20initialized%20during%20declaration.